conans/client/graph/range_resolver.py

```diff
@@ -67,19 +67,17 @@ def _resolve_local(self, search_ref, version_range):
     def _search_and_resolve_remotes(self, search_ref, version_range):
         results = []
         remotes = self._conan_app.enabled_remotes
```

Comment:
Why here it is ``enabled`` and not ``select_remotes``?

Reply:
sorry, that was a mistake, that should be `selected_remotes` 👍